### ISSUE TITLE
Push Parquet 1.9 upgrade upstream

### ISSUE THREAD

**Issue by ash211** [palantir/spark#60]
https://issues.apache.org/jira/browse/SPARK-13127

In this repo it's done across a few commits/PRs:
- initial upgrade: https://github.com/palantir/spark/commit/0ed78ba446d2e74d000d58530f5ce27d2cab9bcd
- followup to close readers: https://github.com/palantir/spark/commit/0ed78ba446d2e74d000d58530f5ce27d2cab9bcd
- open readers only once: https://github.com/palantir/spark/pull/59

**Comment by schlosna** [comment]
Initial upgrade to Parquet 1.8.1-palantir7 was 3449f8467146cb80651245aaf92fef2be52b83ae

**Comment by ash211** [comment]
Activity today in the upstream issue with a new PR opened

**Comment by robert3005** [comment]
They won't pick it up due to avro 1.8 bump. Have to wait for spark 3.0.
